src/ui/screens/WebcamCaptureTool.tsx

```diff
@@ -46,15 +53,34 @@ export default class WebcamCaptureTool extends React.Component<any,any> {
     })
   }
 
+  determinePhotoType(dataUri: string): PhotoAttach {
+    const parts: string[] = dataUri.split(",", 2);
```

Comment:
We should check errors here and default to a certain type, similar to what you did in the verifier side.